site/stripe/DonateForm.tsx

```diff
@@ -75,24 +76,22 @@ export class DonateForm extends React.Component {
 
     @action.bound setPresetAmount(amount?: number) {
         this.presetAmount = amount
-        this.isCustom = false
+        this.customAmount = ""
     }
 
     @action.bound setCustomAmount(amount: string) {
         this.customAmount = amount
-        this.isCustom = true
-    }
-
-    @action.bound setIsCustom(isCustom: boolean) {
-        this.isCustom = isCustom
+        this.presetAmount = undefined
     }
 
     @action.bound setName(name: string) {
```

Comment:
[This is an enormous can of worms](https://www.kalzumeus.com/2010/06/17/falsehoods-programmers-believe-about-names/), but I think we should trim this input. Right now, `" "` is a valid name.